pkg/client/didexchange/client.go

```diff
@@ -41,22 +39,31 @@ type provider interface {
 type Client struct {
 	service.Action
 	service.Message
-	didexchangeSvc           service.DIDComm
+	didexchangeSvc           Service
 	kms                      kms.KeyManager
 	inboundTransportEndpoint string
 	actionCh                 chan service.DIDCommAction
 	msgCh                    chan service.StateMsg
 	connectionStore          *didexchange.ConnectionRecorder
 }
 
+// Service defined DID Exchange service.
+type Service interface {
```

Comment:
Is this interface used outside this package?

Reply:
No, this can be un-exported.

Reply:
[Done](https://github.com/hyperledger/aries-framework-go/compare/b13f7cb1868e9ddbd6425936dec4f1c055d744bb..15d518e3af35c643a0c153e8b799d2bfd14e6e7b#diff-427522b353a28b1ec350c29c79d387b0R51) - renamed to `protocolService` as `service` would conflict with an import.